### DISCUSSION TITLE
Consider including CodeChecker to CDT-Cloud

### DISCUSSION THREAD

**Discussion by MatthewKhouzam** [eclipse-cdt-cloud/cdt-cloud#1] [Ideas]
Code Checker (https://github.com/Ericsson/CodeChecker) is a static analyzer that uses clang-
tidy but also performs data flow and security checks on code. It could be an item to watch.

soon a vs code plug-in will be available:
https://github.com/Ericsson/CodeCheckerVSCodePlugin/releases

**Comment by planger** [comment]
Thanks for sharing that!
CodeChecker indeed looks like a very interesting and already quite popular project. I think it would be an excellent contribution to CDT.Cloud. Imho, we should include the VSCode plug-in in CDT.Cloud Blueprint, independently of where CodeChecker will eventually live.

**Comment by MatthewKhouzam** [comment]
Here is the OpenVSX page https://open-vsx.org/extension/codechecker/codechecker/reviews
